src/app/results/results.component.css

```diff
@@ -613,26 +607,17 @@ div.autocorrect{
   #search-tools {
     padding-left: 30px;
   }
-  #search-options {
-    margin-left: 7%;
-  }
   #search-options li {
     padding: 0px 4vw 12px;
   }
 }
 
 @media screen and (max-width: 460px) {
-  .feed.col-md-10.col-md-offset-1.col-sm-10.col-sm-offset-1.col-lg-10.col-lg-offset-1 {
```

Comment:
Like @nikhilrayaprolu pointed out this is a bad class name, I fixed it in the large screen pull, please wait for it to get merged, then we will have lesser problems.